CodeEdit/Utils/Extensions/Color/Color+HEX.swift

```diff
@@ -31,7 +31,6 @@ extension Color {
         self.init(.sRGB, red: Double(red) / 255, green: Double(green) / 255, blue: Double(blue) / 255, opacity: alpha)
     }
 
-    /// Returns an Int representing the `Color` in hex format (e.g.: 0x112233)
```

Comment:
Why the removal of the comment?

Reply:
Hey guys, sorry. Seems like it just slipped my mind! It's back now